csv_writer.py

```diff
@@ -14,18 +14,18 @@ def site_csv_writer(source_sites, receiving_sites, hv_sites):
     """Create source, receive, high volume receiving CSV file"""
     logging.info('Creating soil source site CSV...')
     with open(constant.SOURCE_CSV_FILE, 'w', encoding='UTF8', newline='') as source_file:
-        writer = csv.DictWriter(source_file, fieldnames=constant.SOURCE_SITE_HEADERS)
+        writer = csv.DictWriter(source_file, fieldnames=constant.SOURCE_SITE_HEADERS, extrasaction='ignore')
```

Comment:
could you please fix the typo for `extraction`

Reply:
This(extrasaction='ignore') is not typo. Please see this instruction:
![image](https://user-images.githubusercontent.com/107894308/212783116-19a9c36d-5ca3-4bd2-a6d4-345105bf0b8e.png)
https://docs.python.org/3.9/library/csv.html#csv-fmt-params

Reply:
thanks for the links